tests/h/accounts/views_test.py

```diff
@@ -465,7 +315,8 @@ def test_post_generates_mail(self,
         reset_mail.assert_called_with(user, "faketoken", "http://example.com")
 
     @mock.patch('h.accounts.views.account_reset_email')
-    def test_post_sends_mail(self, reset_mail, mailer, pyramid_request):
+    def test_post_sends_mail(
+            self, reset_mail, form_validating_to, mailer, pyramid_request):
```

Comment:
It's a small thing, but I don't really understand why half the time you're doing:

```python
def test_something(with,
                   many,
                   arguments):
```

and half the time doing


```python
def test_something(
    with, many, arguments):
```

Can we stick to the former?